src/Robo/Commands/Setup/ToggleModulesCommand.php

```diff
@@ -41,13 +41,16 @@ public function toggleModules($options = [
     }
 
     if (isset($environment)) {
-      // Enable modules.
-      $enable_key = "modules.$environment.enable";
-      $this->doToggleModules('pm-enable', $enable_key);
-
-      // Uninstall modules.
-      $disable_key = "modules.$environment.uninstall";
-      $this->doToggleModules('pm-uninstall', $disable_key);
+      // Determine if modules should be enabled or uninstalled first.
+      $enable_first = $this->getConfigValue("modules.$environment.enable_first", TRUE);
```

Comment:
I'm not crazy about the `modules.$environment.enable_first` config key. I'm thinking maybe we should have an ordered array, in case more commands are added later.

E.g., `setup.toggle.operations: [ 'enable', 'uninstall' ]`

I'm also thinking we may want to move `modules.[env].enable` to `setup.toggle.envs.[env].enable`, etc.